packages/at_lookup/lib/src/at_lookup_impl.dart

```diff
@@ -34,7 +34,6 @@ class AtLookupImpl implements AtLookUp {
 
   late int _rootPort;
 
-  String? privateKey;
```

Comment:
mark this as deprecated instead of removing.if any old clients set private key,  they won't break.

Reply:
updated